src/CameraControls.ts

```diff
@@ -159,6 +157,11 @@ export class CameraControls extends EventDispatcher {
 	protected _needsUpdate = true;
 	protected _updatedLastTime = false;
 
+	protected _dragStartPosition: _THREE.Vector2;
+	protected _lastDragPosition: _THREE.Vector2;
+	protected _dollyStart: _THREE.Vector2;
```

Comment:
these 3 do not need to be private. let's make it as it is

Reply:
i follow the same rule as the other ones by making them protected. Couldn't it be useful for a extended method to be able to access thoses?

Reply:
these 3 are only used in DOM event handlers.
Maybe we could expose them later.

Reply:
@yomotsu ok will change that